### PR TITLE
Add ChapterChapter

### PR DESCRIPTION
**Pull request by brinxmat**
- Add ChapterArticle
- Make JournalArticle subclass of ChapterArticle
- Add test

### CODE REVIEW

File: src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    @DisplayName("ChapterArticle exists")
+    @Test
+    void chapterArticleExists() {
+        new ChapterArticle();
+    }
+
+    @DisplayName("ChapterArticle: objectMapper can deserialize object")
+    @Test
+    void objectMapperReturnsChapterChapterWhenInputJsonIsWellFormed() throws JsonProcessingException {
```
Comment (axthosarouris) [inline src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java:33]: ChapterChapter -> ChapterArticle

File: src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        Pages expectedPages = generatePages(expectedBegin, expectedEnd);
+        String json = generateWellFormedJson(expectedBegin, expectedEnd, true);
+        ChapterArticle chapterArticle = objectMapper.readValue(json, ChapterArticle.class);
+        assertEquals(expectedPages, chapterArticle.getPages());
+        assertTrue(chapterArticle.isPeerReviewed());
+    }
+
+    @DisplayName("ChapterArticle: objectMapper can serialize valid input")
+    @Test
+    void objectMapperReturnsValidJsonWhenInputIsValidChapterChapter() throws JsonProcessingException,
```
Comment (axthosarouris) [inline src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java:45]: ChapterChapter

File: src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ChapterArticleTest {
+
+    private static final ObjectMapper objectMapper = JsonUtils.objectMapper;
+    public static final String CHAPTER_CHAPTER = "ChapterArticle";
```
Comment (axthosarouris) [inline src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java:23]: CHAPTER_CHAPTER

File: src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        return new Range.Builder()
+                .withBegin(begin)
+                .withEnd(end)
+                .build();
+    }
+
+    private String generateWellFormedJson(String begin,
+                                          String end,
+                                          boolean peerReviewed) {
+        return "{\n"
```
Comment (janbj) [inline src/test/java/no/unit/nva/model/instancetypes/ChapterArticleTest.java:100]: I like using Map.of() for this to avoid the messy string concatenation.

**Pull request merged**